docs/guide/troubleshooting.md

```diff
@@ -4,37 +4,6 @@ See [Rollup's troubleshooting guide](https://rollupjs.org/troubleshooting/) for
 
 If the suggestions here don't work, please try posting questions on [GitHub Discussions](https://github.com/vitejs/vite/discussions) or in the `#help` channel of [Vite Land Discord](https://chat.vite.dev).
 
-## CJS
-
-### Vite CJS Node API deprecated
```

Comment:
I wonder if we should keep this section still for a little while as v5 and v6 has a link to this hash in its warnings. Or we could add JS that redirects to v6.vite.dev. But not really a blocker for me either ways.

Reply:
Nice!